src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,103 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int DEFAULT_SIZE_DATA = 10;
+    public static final double INCREASE_BY = 1.5;
+    private int size;
+    private T[] dataType;
```

Comment:
a bit confusing name, maybe it would be better to call it something like dataObject or arrayObject?